### ISSUE TITLE
Hiding the accordion content doesn't kick the animation.

### ISSUE THREAD

**Issue by iFlameing** [eea/volto-accordion-block#30]
When we are hiding the accordion content the closing animation doesn't work.

Fix
I fixed the problem by making the `active` props to true all the time. Because if this prop is false then we are adding the display none property which breaks the Animation.

**Comment by iFlameing** [comment]
@tiberiuichim I am not able to create a pr because I don't have permission. We only need to change the active to true on these lines

https://github.com/eea/volto-accordion-block/blob/master/src/components/manage/Blocks/Accordion/AccordionEdit.jsx#L97
https://github.com/eea/volto-accordion-block/blob/master/src/components/manage/Blocks/Accordion/View.jsx#L83

**Comment by nileshgulia1** [comment]
@iFlameing Adding an `active` class always (active = true) distort the accordion design in a way.
<img width="452" alt="Screenshot 2021-09-27 at 8 40 41 PM" src="https://user-images.githubusercontent.com/22280901/134935864-c02eb685-8683-4636-8998-ea03f976493a.png">
There must be something in the react-animate-height lib we are using. I will look into it.

**Comment by iFlameing** [comment]
@nileshgulia1 I looked into it and I found the solution. Nothing wrong with the Animate Height library. here is the solution.

```
        <AnimateHeight
          animateOpacity
          duration={500}
          height={isExclusive(index) ? 'auto' : 0}
        >
          <Accordion.Content active={true}>{children}</Accordion.Content>
        </AnimateHeight>
        ```
        
        ```
                      <AnimateHeight
                animateOpacity
                duration={500}
                height={isExclusive(index) ? 'auto' : 0}
              >
                <Accordion.Content active={true}>
                  <RenderBlocks
                    {...props}
                    metadata={metadata}
                    content={panel}
                  />
                </Accordion.Content>
              </AnimateHeight>
              ```

**Comment by nileshgulia1** [comment]
@iFlameing The fix is nice. I would still like to use the calculated value of active controller for content.
PR: https://github.com/eea/volto-accordion-block/pull/31

**Comment by nileshgulia1** [comment]
fixed in #31
